GameServerCore/Domain/IStackable.cs

```diff
@@ -5,7 +5,7 @@ public interface IStackable
         int MaxStacks { get; }
         int StackCount { get; }
         bool IncrementStackCount();
-        bool DecrementStackCount();
+        bool DecrementStackCount(int stacksToRemove = 0);
```

Comment:
Decrement is the opposite of Increment, which goes up by one. Instead of using Decrement, use SetStacks and revert the changes to Decrement.

Reply:
Resolved